spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe UsersController do
+  describe '#destroy' do
+    it 'destroys the current user and redirects to sign in page, with a helpful flash message' do
+      sign_in_as_user
+      expect do
+        delete :destroy
+        expect(response).to redirect_to(root_url)
+        expect(flash.now[:success]).to eq t('loa1.cancel.success')
+      end.to change(User, :count).by(-1)
+    end
```

Comment:
I didn't know if the delete inside the block would apply to expectations outside of the block, if that makes sense. thanks!